.gitignore

```diff
@@ -8,3 +8,5 @@
 /tmp/
 
 Gemfile.lock
+
+tools/StepFunctionsLocal/
```

Comment:
The expectation is that the java simulator can be put right in the tools dir, and we can run it directly.

Reply:
Removed